### PR TITLE
[11.0][MIG] account_invoice_line_sort: Migration to 11.0

### PR DESCRIPTION
**Pull request by cristinamartinrod**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Check Travis status.

File: account_invoice_line_sort/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Sort Customer Invoice Lines",
+
+    'summary': """
+        Manage sort of customer invoice lines by customers""",
+
+    'author': "ACSONE SA/NV,Odoo Community Association (OCA)",
+    'website': "http://acsone.eu",
```
Comment (nikul-serpentcs) [inline account_invoice_line_sort/__manifest__.py]: @cristinamartinrod  Replace URL as per [this](https://agithub.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)

File: account_invoice_line_sort/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2015 Acsone
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
```
Comment (nikul-serpentcs) [inline account_invoice_line_sort/models/res_partner.py]: Replace `openerp` to `odoo`

File: account_invoice_line_sort/models/account_invoice.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                                  default='sequence')
+    line_order_direction = fields.Selection(AVAILABLE_ORDER_OPTIONS,
+                                            "Sort Direction",
+                                            default='asc')
+
+    @api.model
+    def get_partner_sort_options(self, partner_id):
+        res = {}
+        if partner_id:
+            p = self.env['res.partner'].browse(partner_id)
```
Comment (nikul-serpentcs) [inline account_invoice_line_sort/models/account_invoice.py]: @cristinamartinrod IMO change Variable name `p` for `partner`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Pull request closed without merging**